contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_network.sh

```diff
@@ -32,7 +32,7 @@ if [ "$netrenamed" = "lan" ]; then
 fi
 
 # Setup IPv6 for the interface
-local ip6addr
+ip6addr
```

Comment:
This looks suspicious. There is no value assignment, so the line should be removed wholly, I think. Plain "ip6addr" sounds like a command...